Makefile.defs

```diff
@@ -395,20 +395,21 @@ ifdef DEBIAN
 	HOST_ARCH:=-mtune=generic -march=native
 endif
 
-# Anything below GCC 6 gets to be forcefully switched to decent standards...
-# NOTE: We're also assuming Clang >= 3.2, but that should more or less hold true on our end.
+# Anything below GCC 6 gets to be forcefully switched to decent standards, because some of our deps require C11 & C++11 support.
+# NOTE: Technically, this also means we require Clang >= 3.2, but that should more or less hold true on our end, because that's truly ancient.
```

Comment:
I don't think older than Clang 5 realistically occurs for us in any scenario.

Reply:
Yeah, that's what I thought. Even the Travis mention a few lines ahead mentions 3.4, and that's already an old comment, for an old Travis image ;).